plugins/outputs/mqtt/mqtt.go

```diff
@@ -31,6 +31,7 @@ type MQTT struct {
 	Database    string
 	Timeout     config.Duration `toml:"timeout"`
 	TopicPrefix string          `toml:"topic_prefix"`
+	Topic       string          `toml:"topic"`
```

Comment:
If you want to deprecate the `TopicPrefix` you should do
```suggestion
	TopicPrefix string          `toml:"topic_prefix deprecated:"1.25.0;use 'topic' instead"`
	Topic       string          `toml:"topic"`
```

Reply:
Fantastic! will add this.